src/main/scala/InsufficientMatingMaterial.scala

```diff
@@ -22,34 +22,14 @@ object InsufficientMatingMaterial {
   }
 
   /*
-   * Determines whether a board position is an automatic draw due to neither player
-   * being able to mate the other as informed by the traditional chess rules.
+   * Determines whether a color (of a game in progress) has mating material
    */
-  def apply(board: Board) = {
-    lazy val kingsAndBishopsOnly = board.pieces forall { p => (p._2 is King) || (p._2 is Bishop) }
-    val kingsAndMinorsOnly = board.pieces forall { p => (p._2 is King) || (p._2 is Bishop) || (p._2 is Knight) }
-
-    kingsAndMinorsOnly && (board.pieces.size <= 3 || (kingsAndBishopsOnly && !bishopsOnOppositeColors(board)))
-  }
-
-  /*
-   * Determines whether a color does not have mating material. In general:
-   * King by itself is not mating material
-   * King + knight mates against king + any(rook, bishop, knight, pawn)
-   * King + bishop mates against king + any(bishop, knight, pawn)
-   * King + bishop(s) versus king + bishop(s) depends upon bishop square colors
-   */
-  def apply(board: Board, color: Color) = {
-
-    val kingsAndMinorsOnlyOfColor = board.piecesOf(color) forall { p => (p._2 is King) || (p._2 is Bishop) || (p._2 is Knight) }
-    lazy val nonKingRolesOfColor = board rolesOf color filter (King !=)
-    lazy val rolesOfOpponentColor = board rolesOf !color
-
-    kingsAndMinorsOnlyOfColor && ((nonKingRolesOfColor toList).distinct match {
-      case Nil => true
-      case List(Knight) => (nonKingRolesOfColor.size == 1) && (rolesOfOpponentColor filter (King !=) filter (Queen !=) isEmpty)
-      case List(Bishop) => !(rolesOfOpponentColor.exists(r => r == Knight || r == Pawn) || bishopsOnOppositeColors(board))
-      case _ => false
-    })
+  def apply(game: Game, color: Color): Boolean = {
```

Comment:
This method is no longer required because of 11151a31be192faaa229eb151a7e6dd391e07bd2.

Reply:
I agree... I suppose this method is complicating the merge, so I'll remove it.

Reply:
Fixed in https://github.com/ddugovic/scalachess/commit/8881550d5bfb08168e46f8bd4c8cea3ef6ae20ca CI test at https://travis-ci.org/ddugovic/scalachess/builds/600357867